doc/langref.html.in

```diff
@@ -10792,6 +10792,30 @@ pub const MAKELOCAL = @compileError("unable to translate C expr: unexpected toke
         <p>{#syntax#}ptr_to_struct_array[index].struct_member{#endsyntax#}</p>
       {#header_close#}
 
+      {#header_open|C Variadic Functions#}
+      <p>Zig supports extern variadic functions.</p>
+      {#code_begin|test|variadic_function#}
+      {#link_libc#}
+      {#code_verbose_cimport#}
+const std = @import("std");
+const testing = std.testing;
+
+pub extern "c" fn printf(format: [*:0]const u8, ...) c_int;
+
+test "variadic function" {
+    try testing.expect(printf("Hello, world!\n") == 14);
+    try testing.expect(@typeInfo(@TypeOf(printf)).Fn.is_var_args);
+}
+      {#code_end#}
+      <p>Non extern variadic functions are not allowed.</p>
```

Comment:
Non-extern variadic functions are an accepted proposal #515

Reply:
I was aware of #515; do you think that I should add a link to the issue?

Reply:
It shouldn't say that they are not allowed, linking the issue and saying it hasn't been implemented would be better.

Reply:
Thanks.

By the way: is it correct to do a `git commit --amend` followed by `git push  --force-with-lease` ?

Reply:
Yes, but you can also just push more commits if you want and then the PR can be squash merged.